cookbook/deployment/workflow/lp_schedules.py

```diff
@@ -1,15 +1,17 @@
 """
 
-Scheduling workflow executions with launch plans
-----------------------------------------------------
+Scheduling Workflow Executions With Launch Plans
+------------------------------------------------
 
 For background on launch plans, refer to :any:`launch_plans`.
 
-For up-to-date documentation on schedules, see the `official docs <https://lyft.github.io/flyte/user/concepts/launchplans_schedules.html#schedules>`_
+Launch plans can be set to run automatically on a schedule if the Flyte platform is properly configured.
+Additionally, the scheduled kick-off time can be used in the workflow as input.
```

Comment:
@katrogan can you remind me one last time - and we'll document here - is this the scheduled time?  or the actual time? (which will be a few seconds off)

Reply:
The former:The time at which the schedule event was triggered.

Reply:
@SandraGH5 could we add this?  so if someone sets a daily schedule for 5am, it'll be exactly 5am, not 5:00:01.1329 or something.

Reply:
Done.